scripts/update-clickup-status.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs');
+const axios = require('axios');
+
+const file = fs.readFileSync(process.env.GITHUB_EVENT_PATH, 'utf8');
```

Comment:
[naming] which file?

Reply:
I'm going to comment the file and implement the missing hooks 😉